Source/Core/DolphinWX/Debugger/AssemblerEntryDialog.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinWX/Debugger/AssemblerEntryDialog.h"
+
+#include <wx/sizer.h>
+#include <wx/stattext.h>
+
+#include "Common/GekkoDisassembler.h"
+#include "Common/StringUtil.h"
+
+AssemblerEntryDialog::AssemblerEntryDialog(u32 address, wxWindow* parent, const wxString& message,
+                                           const wxString& caption, const wxString& value,
+                                           long style, const wxPoint& pos)
+    : m_address(address)
+{
+  Create(parent, message, caption, value, style, pos);
+}
+
+bool AssemblerEntryDialog::Create(wxWindow* parent, const wxString& message,
+                                  const wxString& caption, const wxString& value, long style,
+                                  const wxPoint& pos)
+{
+  // Do not pass style to GetParentForModalDialog() as wxDIALOG_NO_PARENT
+  // has the same numeric value as wxTE_MULTILINE and so attempting to create
+  // a dialog for editing multiline text would also prevent it from having a
+  // parent which is undesirable. As it is, we can't create a text entry
+  // dialog without a parent which is not ideal neither but is a much less
+  // important problem.
+  if (!wxDialog::Create(GetParentForModalDialog(parent, 0), wxID_ANY, caption, pos, wxDefaultSize,
+                        wxDEFAULT_DIALOG_STYLE | wxRESIZE_BORDER))
+  {
+    return false;
+  }
+
+  m_dialogStyle = style;
+  m_value = value;
+
+  wxBeginBusyCursor();
+
+  wxBoxSizer* top_sizer = new wxBoxSizer(wxVERTICAL);
+
+  wxSizerFlags flags_border2;
+  flags_border2.DoubleBorder();
+
+#if wxUSE_STATTEXT
```

Comment:
Was part of this code copied from an example or something? wxStaticText is used quite extensively in the GUI code. There's no need to check if it's available or not.